catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/adapter/test/FilterAdapterTest.java

```diff
@@ -95,25 +96,25 @@
             new DefaultPeriodDuration(DAY_IN_MILLISECONDS);
 
     @Test
-    public void includeFilter() {
+    public void testIncludeFilter() {
```

Comment:
Why did you add test to the name of every test method?  That is not required with the annotation.